Cargo.toml

```diff
@@ -1,14 +1,186 @@
 [workspace]
 members = [
-    "crates/*",
+    # EDR
+    "crates/edr_defaults",
+    "crates/edr_eth",
+    "crates/edr_napi",
+    "crates/edr_provider",
+    "crates/edr_solidity",
+    "crates/edr_test_utils",
+    "crates/tools",
+    # Foundry
+    "crates/foundry/cheatcodes",
+    "crates/foundry/cheatcodes/spec",
+    "crates/foundry/common",
+    "crates/foundry/config",
+    "crates/foundry/evm/core",
+    "crates/foundry/evm/coverage",
+    "crates/foundry/evm/evm",
+    "crates/foundry/evm/fuzz",
+    "crates/foundry/evm/traces",
+    "crates/foundry/forge",
+    "crates/foundry/macros",
+    "crates/foundry/test-utils",
 ]
 resolver = "2"
 
+[workspace.package]
+version = "0.3.8"
+edition = "2021"
+
 [profile.dev]
 rpath = true
+# Speed up compilation time for dev builds by reducing emitted debug info.
+# NOTE: Debuggers may provide less useful information with this setting.
+# Uncomment this section if you're using a debugger.
+debug = 1
 
+# Optimized release profile.
 [profile.release]
 rpath = true
+opt-level = 3
+debug = "line-tables-only"
+lto = "fat"
+strip = "debuginfo"
+panic = "abort"
+codegen-units = 1
 
 [profile.test]
 opt-level = 2
+
+# Speed up tests and dev build.
+[profile.dev.package]
+# Solc and artifacts
+foundry-compilers.opt-level = 3
+solang-parser.opt-level = 3
+lalrpop-util.opt-level = 3
+serde_json.opt-level = 3
+
+# EVM
+alloy-dyn-abi.opt-level = 3
+alloy-json-abi.opt-level = 3
+alloy-primitives.opt-level = 3
+alloy-sol-type-parser.opt-level = 3
+alloy-sol-types.opt-level = 3
+hashbrown.opt-level = 3
+keccak.opt-level = 3
+revm-interpreter.opt-level = 3
+revm-precompile.opt-level = 3
+revm-primitives.opt-level = 3
+revm.opt-level = 3
+ruint.opt-level = 3
+sha2.opt-level = 3
+sha3.opt-level = 3
+tiny-keccak.opt-level = 3
+bitvec.opt-level = 3
+
+# fuzzing
+proptest.opt-level = 3
+foundry-evm-fuzz.opt-level = 3
+
+# keystores
+scrypt.opt-level = 3
+
+[workspace.dependencies]
```

Comment:
Is there a follow-up to make the EDR crates use the workspace dependencies?

Reply:
Yep it's here: https://github.com/NomicFoundation/edr/issues/467